themes/digital.gov/layouts/partials/core/pagination.html

```diff
@@ -112,7 +112,7 @@
 
       <!-- Output page numbers. -->
       {{ if eq ($.Scratch.Get "page_number_flag") true }}
-        <li class="page{{ if eq . $paginator }}active{{ end }}">
+        <li class="page {{ if eq . $paginator }}active{{ end }}">
```

Comment:
Fixed class concatenation bug to allow for correct pagination active page styles.